src/theory/quantifiers/sygus/cegis_unif.cpp

```diff
@@ -36,43 +36,76 @@ bool CegisUnif::initialize(Node n,
                            std::vector<Node>& lemmas)
 {
   Trace("cegis-unif") << "Initialize CegisUnif : " << n << std::endl;
-  Assert(candidates.size() > 0);
-  if (candidates.size() > 1)
+  /* For regular CEGIS */
+  d_base_body = n;
+  if (d_base_body.getKind() == NOT && d_base_body[0].getKind() == FORALL)
   {
-    return false;
+    for (const Node& v : d_base_body[0][0])
+    {
+      d_base_vars.push_back(v);
+    }
+    d_base_body = d_base_body[0][1];
   }
-  d_candidate = candidates[0];
-  Trace("cegis-unif") << "Initialize unif utility for " << d_candidate
-                      << "...\n";
-  d_sygus_unif.initialize(d_qe, candidates, d_enums, lemmas);
-  Assert(!d_enums.empty());
-  Trace("cegis-unif") << "Initialize " << d_enums.size() << " enumerators for "
-                      << d_candidate << "...\n";
-  /* initialize the enumerators */
-  for (const Node& e : d_enums)
+  /* Init UNIF util */
+  /* TODO combine with enumeration manager */
+  if (false)
```

Comment:
Remove this.  You should make this code do the right thing (if necessary, modify the SygusUnifRL utility to not allocate things you can't handle).